cloud/docker/docker_compose

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+
+# (c) 2015, Sam Yaple <sam@yaple.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: docker_compose
+version_added: 1.8.4
+short_description: Manage docker containers with docker-compose
+description:
+     - Manage the lifecycle of groups of docker containers with docker-compose
+options:
+  command:
+    description:
+      - Select the compose action to perform
+    required: true
+    choices: ['build', 'kill', 'pull', 'rm', 'scale', 'start', 'stop', 'restart', 'up']
```

Comment:
In this case though, 'state' isn't really accurate. state=scale doesn't make much sense to me. state=up and state=start are two different commands and I think it would lead to confusion.